dnf/cli/format.py

```diff
@@ -15,8 +15,23 @@
 # Red Hat, Inc.
 
 from __future__ import unicode_literals
+from datetime import datetime
```

Comment:
I don't see how past mistakes can excuse your current mistakes... If someone breaks a law, it doesn't mean that everybody can break it.